framework/src/governance/lib/data-catalog-database.ts

```diff
@@ -59,6 +114,8 @@ export class DataCatalogDatabase extends TrackedConstruct {
 
     super(scope, id, trackedConstructProps);
     const catalogType = this.determineCatalogType(props);
+    this.permissionModel = props.permissionModel || DataCatalogDatabase.DEFAULT_PERMISSION_MODEL;
+    const useLakeFormation = props.permissionModel === PermissionModel.LAKE_FORMATION || props.permissionModel === PermissionModel.HYBRID;
```

Comment:
can you first if the value exist? instead of this being implicit. Meaning use `props?.`